lib/JSON/Validator/Formats.pm

```diff
@@ -31,8 +31,10 @@ sub check_date {
 }
 
 sub check_date_time {
-  my @dt = $_[0] =~ m!^(\d{4})-(\d\d)-(\d\d)[T ](\d\d):(\d\d):(\d\d(?:\.\d+)?)(?:Z|([+-])(\d+):(\d+))?$!io;
+  my @dt = $_[0] =~ m!^(\d{4})-(\d\d)-(\d\d)[T ](\d\d):(\d\d):(\d\d(?:\.\d+)?)(?:Z|([+-])(\d\d):(\d\d))?$!io;
   return 'Does not match date-time format.' unless @dt;
+  return 'Time offset hour out of range.' if defined $dt[7] and $dt[7] > 23;
+  return 'Time offset minute out of range.' if defined $dt[8] and $dt[8] > 59;
```

Comment:
I don't understand why this is added. Isn't already covered in https://github.com/mojolicious/json-validator/blob/master/t/jv-formats.t#L29-L30 ?